src/Map/src/Bridge/Google/src/Option/ControlPosition.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Bridge\Google\Option;
+
+/**
+ * Identifiers used to specify the placement of controls on the map.
+ * Controls are positioned relative to other controls in the same layout position.
+ * Controls that are added first are positioned closer to the edge of the map.
+ * Usage of "logical values" (see https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) is recommended
+ * in order to be able to automatically support both left-to-right (LTR) and right-to-left (RTL) layout contexts.
```

Comment:
```suggestion
```

Reply:
Whu?

Reply:
- "logical values is recommended" -> nope we made a choice
- It feels documentation that will be no more read than maintained but i can be wrong here

Reply:
Let's remove it